pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -1859,6 +1859,53 @@ AddressFieldError:
       description: the address field, where `address` means the general address instead of a single field
       example: street1
 
+#####
+#
+# Invitations
+#
+#####
+Invitation:
+  type: object
+  required:
+    - invitationType
+    - invitationId
+    - createdAt
+    - voidedAt
+    - verifiedAt
+    - acceptedAt
+    - notes
+  properties:
+    invitationType:
+      description: the type of invitation
+      type: string
+      enum:
+        - AGE_UP
+        - RECRUITMENT
+      example: RECRUITMENT
+    invitationId:
+      description: the id of the invitation
+      type: string
+    createdAt:
+      description: timestamp of when invitation was created, in ISO-8601 UTC format
+      type: string
+      example: 2020-05-13T17:58:33.887Z
+    voidedAt:
+      description: timestamp of when invitation is voided, in ISO-8601 UTC format
+      type: string
+      nullable: true
+    verifiedAt:
+      description: timestamp of when invitation is verified, in ISO-8601 UTC format
+      type: string
+      nullable: true
+    acceptedAt:
+      description: timestamp of when invitation is accepted, in ISO-8601 UTC format
+      type: string
+      nullable: true
+    notes:
+      description: the notes for the invitation
```

Comment:
Where are the notes coming from? Are the researcher, provided by the user, etc? What's the context here?

Reply:
The `notes` are meant to be a place where study staff can write arbitrary notes.

Reply:
I wasn't looking for an answer for my own knowledge, that's something that should be in the docs.

Reply:
Yup, that's understood. I simply have the tendency to respond and engage in GitHub comments.

Reply:
Thinking more about this, it doesn't seem the `notes` property will be useful or appropriate in the participant-visible API. I have removed it and only added it in the admin side of things.

Reply:
That is even better! It seems like we might want to consider splitting the API docs out into client-facing, and management-facing.